### PR TITLE
Uncomment Immovable Rod

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

Part of the important gameplay loop for engineering is that they're supposed to be repairing the station when shit breaks it, so the ImmovableRod event is a key part of the game. However, it was currently commented out because people didn't like it gibbing characters with no reactability? Well news flash for people, there's a datafield for that. I can also directly control how much damage it deals.

So in exchange for uncommenting the Immovable Rod event, I've set the immovable rod spawned to be a little fairer for players, while still being able to cause damage to the station that will require repairs(And also expose players to vacuum. So that it puts them in danger.

# Changelog

:cl:
- add: Immovable Rod Event has returned
- add: Immovable Rod Event has been made more fair. It won't automatically gib people, just crit them with a ton of blunt damage(That you can survive by wearing good armor), or just being an Oni funnily enough.

### CODE REVIEW

File: Resources/Prototypes/GameRules/events.yml [hunk 1]
```diff
@@ -517,18 +517,18 @@
...
-#  noSpawn: true
-#  components:
-#  - type: StationEvent
-#    startAnnouncement: true
-#    weight: 5
-#    duration: 1
-#    earliestStart: 45
-#    minimumPlayers: 20
-#  - type: ImmovableRodRule
+- type: entity # DeltaV - Why does this exist??
```
Comment (stellar-novas) [inline Resources/Prototypes/GameRules/events.yml:520]: Remove comment?

**Review by stellar-novas — APPROVED** [review]

**Comment by Remuchi** [comment]
MobMimic once again failing tests pepeLaugh

**Comment by DEATHB4DEFEAT** [comment]
image

This seems like a pretty big problem, we can't reconnect the station once split.

**Comment by Remuchi** [comment]
> image
> 
> This seems like a pretty big problem, we can't reconnect the station once split.

Isn't the rod in the event doesn't break tiles?
```
ImmovableRodKeepTilesStill
```

**Review by FoxxoTrystan — APPROVED** [review]

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**